src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -243,11 +245,33 @@ public void AddTagHelperAttribute(string name, object value)
             Output.Reinitialize(tagName, tagMode);
         }
 
+        /// <summary>
+        /// A delegate used to execute children asynchronously with the pages <see cref="HtmlEncoder" /> in scope and
+        /// set <see cref="Output"/>'s <see cref="TagHelperOutput.Content"/> to the rendered results.
```

Comment:
Describe what this method does, not some delegate somewhere that happens to refer to it.